solidity/contracts/system/KeepFactorySelection.sol

```diff
@@ -34,28 +38,38 @@ library KeepFactorySelection {
 
         KeepFactorySelector factorySelector;
 
-        // Standard ECDSA keep factory: KEEP stake and ETH bond.
+        // Standard ECDSA keep vendor and factory: KEEP stake and ETH bond.
         // Guaranteed to be set for initialized factory.
+        IBondedECDSAKeepVendor keepStakeVendor;
```

Comment:
`keepStakeFactoryVendor` maybe? We are not vendoring keeps, we are vendoring factories.

Reply:
In keep-ecdsa it is called just `vendor` not `factory vendor`.

Reply:
So maybe `keepStakedVendor` and `keepStakedFactory`? I still don't love it but I think it's a little better, especially that we have `fullyBackedVendor` and `fullyBackedFactory`.

Reply:
57274307025e2ba52329e3ae112cae4888653c2d